rmgpy/data/solvation.py

```diff
@@ -138,7 +138,7 @@ def processOldLibraryEntry(data):
     raise NotImplementedError()
 
 
-class SolventData():
+class SolventData(object):
```

Comment:
Is there a particular reason for this?

Reply:
Since Python 2.3, fixes/features for classes have only been implemented for new-style classes, so it's generally bad practice to continue using old-style classes. Figured I'd fix it here.